src/models/dish.ts

```diff
@@ -0,0 +1,63 @@
+import {Document, Schema, model, Types} from "mongoose";
+
+export interface IDish extends Document{
+    name: string;
+    cooker: Types.ObjectId;
+    description: string;
+    images: string[];
+    quantity: number;
+    price: number;
+    category: "vegeterian" | "non-vegeterian" | "gluten-free" | "vegan";
+    sold? : boolean;
+    slug? : string;
```

Comment:
slug will be computed internally so i think it should not be optional.
` 
 slug: string;
`